.github/workflows/test-all-packages.yml

```diff
@@ -267,6 +267,9 @@ jobs:
       - name: yarn test (fast-usdc)
         if: (success() || failure())
         run: cd packages/fast-usdc && yarn ${{ steps.vars.outputs.test }} | $TEST_COLLECT
+      - name: yarn test (fast-usdc-contract)
+        if: (success() || failure())
+        run: cd packages/fast-usdc-contract && yarn ${{ steps.vars.outputs.test }} | $TEST_COLLECT
```

Comment:
This name could change as part of https://github.com/Agoric/agoric-sdk/pull/11137